### PR TITLE
Update README

### LINKED ISSUE #1043: Review README for 0.2
We should for sure upgrade the example, but also quickly reconsider the content:

https://github.com/qiboteam/qibolab/blob/9bbbbe8c30bad4e99ba2cbbcb32ecb63147a43d4/README.md?plain=1#L15
routines are not here since ever...

https://github.com/qiboteam/qibolab/blob/9bbbbe8c30bad4e99ba2cbbcb32ecb63147a43d4/README.md?plain=1#L16
we currently have no emulator in 0.2

https://github.com/qiboteam/qibolab/blob/9bbbbe8c30bad4e99ba2cbbcb32ecb63147a43d4/README.md?plain=1#L14
I'm not completely sure what does "heterogeneous" stands for, in this context (and what is adding wrt the point directly above it)

### PR DESCRIPTION
**Pull request by stavros11**
Close #1043. I also removed a `log` message from `platform.connect` because it would trigger very long output if the default dataclass (pydantic) representation is used for the instrument object (which has been overlooked mainly due to QM hiding the logger).

This should be ready, but maybe further changes will be needed if we do any last minute tweaks in the interface.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,38 +42,74 @@ options = ExecutionParameters(nshots=1000)
 results = platform.execute([sequence], options)
```
Comment (alecandido) [inline README.md:42]: If #1052 is approved:
```suggestion
results = platform.execute([sequence], nshots=1000)
```

File: README.md [hunk 2]
```diff
@@ -26,43 +26,14 @@ A simple example on how to connect to a platform and use it execute a pulse sequ
 
 ```python
 from qibolab import create_platform, ExecutionParameters
```
Comment (alecandido) [inline README.md:28]: If #1052 is approved
```suggestion
from qibolab import create_platform
```

**Review by alecandido — APPROVED** [review]
I flagged what I believe may require updates with the remaining PRs.

However, whenever everything else is merged, we can merge even this.

File: README.md [hunk 3]
```diff
@@ -71,38 +42,74 @@ options = ExecutionParameters(nshots=1000)
 results = platform.execute([sequence], options)
 
 # Print the acquired shots
-print(results.samples)
+readout_id = sequence.acquisitions[0][1].id
```
Comment (scarrazza) [inline README.md:45]: I think we should explain the meaning of [0][1] and `.id` here.

File: README.md [hunk 4]
```diff
@@ -10,9 +10,9 @@ the automatic deployment of quantum circuits on quantum hardware.
 Some of the key features of Qibolab are:
 
 - Deploy Qibo models on quantum hardware easily.
-- Create custom experimental drivers for custom lab setup.
+- Create experimental drivers for custom lab setup.
 - Support multiple heterogeneous platforms.
-- Use existing calibration procedures for experimentalists.
+- Use calibration procedures from [Qibocal](https://github.com/qiboteam/qibocal).
```
Comment (scarrazza) [inline README.md:15]: Could you please port these changes to the sphinx docs too?

File: README.md (hunk 1 above)
Comment (stavros11) [reply README.md]: Done in ee0b9d07478a45dd9a1ab159bb8d1f82f4d431ec, thanks.

File: README.md (hunk 2 above)
Comment (stavros11) [reply README.md]: ee0b9d07478a45dd9a1ab159bb8d1f82f4d431ec

File: README.md (hunk 3 above)
Comment (stavros11) [reply README.md:48]: Done in aa113bffe399ed3bb581bcda2e9cbd5b45db59bc.

File: README.md (hunk 4 above)
Comment (stavros11) [reply README.md:15]: Done in b5f8db4a169c81e35dcfb4437135efec81e767da.

**Review by scarrazza — APPROVED** [review]

**Pull request merged**